website/blog/2018-01-10-1.10.0.md

```diff
@@ -0,0 +1,76 @@
+---
+author: Christopher Chedeau (@vjeux)
+authorURL: https://twitter.com/vjeux
+title: "Prettier 1.10: One Year of Prettier 🎉"
+---
+
+Happy Prettier-versary!
+
+This release adds partial support for Vue ([#3563]), fixes the printing of Flow type annotation comments ([#3449]), and fixes a bug with comments in SCSS selectors ([#3649])
```

Comment:
I feel like these are the major changes (should the plugin API be publicized in this release?), but feel free to change this if you feel that some of these aren’t important or something important is missing.